cosmos/operators/local.py

```diff
@@ -248,6 +251,12 @@ def calculate_openlineage_events_completes(
         for key, value in env.items():
             os.environ[key] = str(value)
 
+        lineage_namespace = os.getenv("OPENLINEAGE_NAMESPACE", "default")
```

Comment:
perhaps the namespace here could be cosmos?

Reply:
@harel we already set the producer URL to be cosmos-specific, would it also make sense to segregate the namespace depending on the tool used to process the data? I honestly don't know best lineage best practices and will follow what you recommend

Reply:
Ideally, the namespace is a logic grouping of the data itself.
I would encourage the users to define that with the ENV_VARS, but I think that if by default it shows up as cosmos, it helps users have a better understanding of where it's coming from and then change as this see fit.

Reply:
producer URL is pretty much internal implementation detail for the OL consumer

Reply:
Changed to `cosmos`, @harels !